src/CharacterStates/Normal.cpp

```diff
@@ -0,0 +1,24 @@
+//
+// Created by lisandro on 25/10/20.
+//
+
+#include "../Player.h"
+#include "Normal.h"
+#include "Running.h"
+#include "Jumping.h"
+
+
+Normal::Normal(int currentFrame, int frameAmount) : CharacterState(currentFrame, frameAmount) {
+}
+
+void Normal::move(const Uint8 *currentKeyStates, Player* player) {} //Nothing to do
```

Comment:
Nop, do not override if you are not going to add anything of value into the code. Confusing for future cases.